src/interface.jl

```diff
@@ -904,9 +904,26 @@ macro default_lie_group_fallbacks(TG, TP, TV, gfield::Symbol, Xfield::Symbol)
             return k
         end
 
+        function LieGroups.exp(G::$TG, X::$TV)
```

Comment:
This shouldn't be needed in this macro (the ManifoldsBase one should cover exp and log).

Reply:
ManifoldsBase never covers the case with one argument, that is a new variant here in LieGroups.

Reply:
eh 2 arguments, so the one without a point I mean.

Reply:
So with Identity I did not run in the 6 ambiguities above but about 200 others :D

Reply:
Or phrased differently, with `:Identity` we basically need a split like for `compose` so a `_compose` / `_exp`. Then this is really just easier, since without the `t` now this is always completely clear what is meant for `exp(M, p, X)` vs `exp(G, X)`, so no semantic ambiguity – for the types see above, I think those would even appear in ManifoldsBase if someone has a type wrapper and specialises on size